src/tiled/commandmanager.h

```diff
@@ -57,8 +67,15 @@ public slots:
 
     CommandManager();
 
+    /**
+     * Updates mActions QList
+     */
```

Comment:
Please don't write comments about things that are immediately obvious when reading the code.